apache-maven/src/main/assembly/component.xml

```diff
@@ -29,6 +29,7 @@ under the License.
     <dependencySet>
       <useProjectArtifact>false</useProjectArtifact>
       <outputDirectory>lib</outputDirectory>
+      <outputFileNameMapping>${artifact.artifactId}-${artifact.version}.${artifact.extension}</outputFileNameMapping>
```

Comment:
This is to have a cleaner and consistent naming of jars in LICENSE file and in 'lib' directory.
We should add 'classifier' in both or drop it in both, actually it seems simpler to do drop

Reply:
I don't like this: why do you find it useful?

Reply:
In order to make the names of the jar files look the same as we write them in the LICENSE file.

I can try to change the LICENSE file to include the qualifier and revert this line

Reply:
reverted